editor/src/components/menubar/menubar.tsx

```diff
@@ -355,6 +359,12 @@ export const Menubar = React.memo(() => {
           <Tile style={{ marginTop: 12, marginBottom: 12 }} size='large'>
             <a onClick={onTriggerResizeTest}>P R</a>
           </Tile>
+          <Tile style={{ marginTop: 12, marginBottom: 12 }} size='large'>
+            <a onClick={onTriggerRegularHighlightTest}>PRH</a>
```

Comment:
love the new mistery buttons! :D

Reply:
I was cursing you whilst adding them :grinning: